google/cloud/iam/samples/BUILD

```diff
@@ -0,0 +1,37 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Examples for the Cloud Pub/Sub C++ client library."""
+
+package(default_visibility = ["//visibility:public"])
+
+licenses(["notice"])  # Apache 2.0
+
+load(":iam_client_samples.bzl", "iam_client_samples")
+
+[cc_test(
+    name = test.replace("/", "_").replace(".cc", ""),
+    timeout = "long",
+    srcs = [test],
+    copts = ["-I$(GENDIR)"],
```

Comment:
Is this needed? With this flag, I was expecting to see the `iam_samples.cc` file including some `.pb.h` file (or similar), but I didn't. Is this needed for some other reason, or can this flag be removed?